test/fixtures/code_samples/_main.java

```diff
@@ -0,0 +1,20 @@
+package test.fixtures.code_samples;
```

Comment:
Also, could you add a block comment on this file, for tests purpose ?

Reply:
@JoaoDanielRufino I am not sure if i entirely got you. You want me to make line 1 as below?
```
/*package test.fixtures.code_samples;*/
```

Reply:
Okay it made sense to me now. I have pushed it